### PR TITLE
Fix typos in lib/debugger

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: lib/debugger/test/int_SUITE_data/my_lists.erl [hunk 1]
```diff
@@ -222,7 +222,7 @@ flatten([], [H|Cont], Tail) ->
 flatten([], [], Tail) ->
     Tail.
 
-%% flat_length(List) (undocumented can be rmove later)
+%% flat_length(List) (undocumented can be remove later)
```
Comment (bjorng) [inline lib/debugger/test/int_SUITE_data/my_lists.erl:225]:
```suggestion
%% flat_length(List) (undocumented can be removed later)
```

File: lib/debugger/test/int_SUITE_data/my_lists.erl [hunk 2]
```diff
@@ -683,7 +683,7 @@ flatten_1([], [H|Cont], Tail) ->
 flatten_1([], [], Tail) ->
     Tail.
 
-%% flat_length(List) (undocumented can be rmove later)
+%% flat_length(List) (undocumented can be remove later)
```
Comment (bjorng) [inline lib/debugger/test/int_SUITE_data/my_lists.erl:686]:
```suggestion
%% flat_length(List) (undocumented can be removed later)
```

File: lib/debugger/test/int_SUITE_data/lists1.erl [hunk 3]
```diff
@@ -221,7 +221,7 @@ flatten([], [H|Cont], Tail) ->
 flatten([], [], Tail) ->
     Tail.
 
-%% flat_length(List) (undocumented can be rmove later)
+%% flat_length(List) (undocumented can be remove later)
```
Comment (bjorng) [inline lib/debugger/test/int_SUITE_data/lists1.erl:224]:
```suggestion
%% flat_length(List) (undocumented can be removed later)
```

File: lib/debugger/test/int_SUITE_data/my_lists.erl [hunk 4]
```diff
@@ -1147,7 +1147,7 @@ flatten_2([], [H|Cont], Tail) ->
 flatten_2([], [], Tail) ->
     Tail.
 
-%% flat_length_2(List) _2(undocumented can be rmove later)
+%% flat_length_2(List) _2(undocumented can be remove later)
```
Comment (bjorng) [inline lib/debugger/test/int_SUITE_data/my_lists.erl:1150]:
```suggestion
%% flat_length_2(List) _2(undocumented can be removed later)
```

File: lib/debugger/test/int_SUITE_data/my_lists.erl [hunk 5]
```diff
@@ -1609,7 +1609,7 @@ flatten_3([], [H|Cont], Tail) ->
 flatten_3([], [], Tail) ->
     Tail.
 
-%% flat_length_3(List) _3(undocumented can be rmove later)
+%% flat_length_3(List) _3(undocumented can be remove later)
```
Comment (bjorng) [inline lib/debugger/test/int_SUITE_data/my_lists.erl:1612]:
```suggestion
%% flat_length_3(List) _3(undocumented can be removed later)
```

File: lib/debugger/test/int_eval_SUITE.erl [hunk 6]
```diff
@@ -89,7 +89,7 @@ bifs_outside_erlang(Config) when is_list(Config) ->
     ok = spawn_eval(Fun),
     ok.
 
-%% Try evalutate spawn_link/3.
+%% Try evaluate spawn_link/3.
```
Comment (bjorng) [inline lib/debugger/test/int_eval_SUITE.erl:92]:
```suggestion
%% Try evaluating spawn_link/3.
```

**Review by bjorng — CHANGES_REQUESTED** [review]

**Comment by kianmeng** [comment]
@bjorng Done, and commits squashed.

File: lib/debugger/test/dbg_ui_SUITE_data/manual_data/src/lists1.erl [hunk 7]
```diff
@@ -221,7 +221,7 @@ flatten([], [H|Cont], Tail) ->
 flatten([], [], Tail) ->
     Tail.
 
-%% flat_length(List) (undocumented can be rmove later)
+%% flat_length(List) (undocumented can be remove later)
```
Comment (bjorng) [inline lib/debugger/test/dbg_ui_SUITE_data/manual_data/src/lists1.erl:224]:
```suggestion
%% flat_length(List) (undocumented can be removed later)
```

**Review by bjorng — CHANGES_REQUESTED** [review]
Thanks. I found one more "remove" that should be changed to "removed" that I missed last time I reviewed.

**Review by bjorng — APPROVED** [review]

**Comment by bjorng** [comment]
Thanks!

**Pull request merged**